client/src/helpers/qnAHelper.js

```diff
@@ -63,89 +63,143 @@ module.exports = {
     return newArr;
 
   },
+//HOLY MOLY - this function got crazy - needed to refactor because it was duplicating answers displayed after sort
+//now it seems to function properly.  Sorts answers by seller and sellerhelpfulness, then sorts remaining user answers
+//based on helpfulness
+  sortAnswers(answers) {
```

Comment:
this is what i was talking about - my sort answers function has really bad time complexity.  Would an API request for answers be more efficient? something to look into for optimization.